easybuild/easyconfigs/w/WRF/WRF-4.6.1-foss-2024a-dmpar.eb

```diff
@@ -0,0 +1,44 @@
+name = 'WRF'
+version = '4.6.1'
+buildtype = 'dmpar'
+versionsuffix = '-%s' % buildtype
+
+homepage = 'https://www.mmm.ucar.edu/models/wrf'
+description = """The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale
+ numerical weather prediction system designed to serve both operational forecasting and atmospheric
+ research needs."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+toolchainopts = {'opt': False}  # don't use agressive optimization, stick to -O2
+
+github_account = 'wrf-model'
+source_urls = [GITHUB_RELEASE]
+sources = ['v%(version)s.tar.gz']
+patches = ['WRF-4.6.1_netCDF-Fortran_separate_path.patch']
+
+checksums = [
+    {'v4.6.1.tar.gz': 'b8ec11b240a3cf1274b2bd609700191c6ec84628e4c991d3ab562ce9dc50b5f2'},
+    {'WRF-4.6.1_netCDF-Fortran_separate_path.patch':
```

Comment:
I don't think this patch is necessary any more with WRF 4.5.2 plus, you can just use preconfigopts:
```
preconfigopts = "export NETCDF=$EBROOTNETCDFMINFORTRAN && export NETCDF_C=$EBROOTNETCDF &&"
```
@asalois-msu Can you sync here, I know you also have working recipes for these versions.

Reply:
Yes, the solutions works. Now, testing for WPS...